### DISCUSSION TITLE
Error in using Kart with QGIS

### DISCUSSION THREAD

**Discussion by fkshuvo** [koordinates/kart#613] [QGIS] [answered]
Hi altruist,

I am having an issue when trying to import a shape file/geopackage data into a repository while using QGIS.
Everytime, it showing the following error:

![image](https://user-images.githubusercontent.com/55429886/165448406-0e065ed0-2436-4696-a957-d083e586d3b1.png)

Kart failed with the following message: 
Error: Please tell me who you are.
Run
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
to set your account's default identity.
Omit --global to set the identity only in this repository.
(Kart uses the same credentials and configuration as git) 

Can anyone help with the issue please?

best regards,

Faysal

**Comment by rcoup** [comment, marked as answer]
Hi Faysal,

My guess is you installed Kart through the QGIS plugin, and we're not correctly setting up your user details. If you open a command prompt (Windows 10: _Start... Windows System.... Command Prompt_) then enter the following, substituting in your own name/email:

```console
kart config --global user.name "Your Name"
kart config --global user.email "your@email.address"
```

These commands setup the name & email for use in Kart repository commit histories. After that I think you'll find it works properly in QGIS.

Please let me know — if this is the case then I'll open a bug to check on the install/configuration.

**Reply by fkshuvo** [reply]
Thats brilliant Robert :) Yes it worked.
